### ISSUE TITLE
Dynamo Player_Automatic opening report window?

### ISSUE THREAD

**Issue by MauroSabiu** [BadMonkeysTeam/Mandrill#28]
Hi Konrad, first of all, great package. 
In my view, it would be really interesting to allow the automatic opening of the report window in order to visualize the charts directly in revit through dynamo player. What do you think?
it could be nice also if related to the functionality of the Data-Shape package. 
Thanks!

**Comment by ksobon** [comment]
@MauroSabiu thanks for sharing. What's the Data Shape package? What does it do? Care to post some examples? I am not familiar with it. 

What you are asking for i can see being possible with a simple `Boolean` input into the Window node that would launch it. I can do that instead of the Button on the node itself. I just wanted to avoid extra inputs if necessary.

**Comment by MauroSabiu** [comment]
@ksobon that would be just great! it should allow to open the window with dynamo closed, exactly what i would like to achieve.
The Data-Shape package allow the user to create user interface windows simple to use also for not-dynamo users. This is the website https://data-shapes.net/
Example:
![image](https://cloud.githubusercontent.com/assets/16916318/21857268/cb0f705a-d81c-11e6-9fbe-650eedbfc4ed.png)

in this case for example i am using your ArchiLab Package in dynamo, but running it with dynamo player and thanks to the Data-Shape package i can select the sheets and the revision in a classic window

**Comment by MauroSabiu** [comment]
here the .dyn file for you information
[Add-Revisions_Test_MS.zip](https://github.com/BadMonkeysTeam/Mandrill/files/699481/Add-Revisions_Test_MS.zip)

**Comment by ksobon** [comment]
@MauroSabiu so I had a look at this and it looks like this will not be possible. What happens is that Mandrill window even when launched via Dynamo Player (or Data Shapes inputs) still very much "belongs" to Dynamo context. Closing Dynamo will close Mandrill automatically. There is no control over that as far as I can see. Data Shapes is able to display a window dialog like that because it actually interrupts the execution of the graph and halts it until you hit "Set Values". Mandrill doesn't behave like that since it's a fully functioning node model, it's execution is asynchronous. Also, since Dynamo's UI is never displayed so will Mandrill's window not be launched. I will ask the Dynamo dev team for any advice, but for now, It doesn't look too good. cc: @DanielHurtubise

**Comment by andydandy74** [comment]
I think this is a feature request for Data-Shapes, not for Mandrill, so maybe it should be posted there:
https://github.com/MostafaElAyoubi/Data-shapes/issues
A sensible approach would be to export the Mandrill report to HTML and the have a new UI component in Data-Shapes that can display HTML. Technically, it is perfectly possible to use a windows form to display HTML (see https://msdn.microsoft.com/en-us/library/system.windows.forms.webbrowser(v=vs.110).aspx). This is actually on a list of contributions I am planning to make to Data-Shapes, though not on the top of the list right now, but I definitely need this feature as well.
CC @MostafaElAyoubi

**Comment by ksobon** [comment]
@andydandy74 i disagree. It would be perfectly sensible for DynamoPlayer not to swallow Mandrill's window. I want to ask Dynamo Player devs if there is a way to exclude certain windows when they hide them (I assume that's how they get rid of Dynamo UI when executing graphs). 
Bypassing this functionality and trying to display HTML code in DataShapes is unnecessary workaround as it should just work with Mandrill.
